react/Analytics.tsx

```diff
@@ -1,33 +0,0 @@
-import { Component } from 'react'
```

Comment:
Isn't this a breaking change?

Reply:
I just moved it to the `components`folder

Reply:
But if it isn't on the root of the react folder it won't be exported. Better check if someone was using it, I've bet @klzns  knows more about it.

Reply:
Oh, I see... I think it's only used here but will check that! Thanks 🙏

Reply:
Moved to the root 🤙